include/json/value.h

```diff
@@ -534,6 +534,7 @@ Json::Value obj_value(Json::objectValue); // {}
   /** \brief Remove the named map member.
 
       Update 'removed' iff removed.
+      'removed' can be NULL if removed value is negligible.
```

Comment:
The word "negligible" is misused here as it implies a small size.
The removed value might be ignorable, but not because of its size.